chemreg/compound/tests/test_filters.py

```diff
@@ -1,24 +1,42 @@
 import pytest
 
 
+@pytest.mark.parametrize("search_type", ["V3000", "V2000", "SMILES"])
```

Comment:
This is slick.  I'm going to start writing my tests to include params like this.